HW8/Game/src/Character.java

```diff
@@ -1,46 +1,54 @@
 public class Character {
-    String name;
-    int health;
-    int strength;
-    int defence;
-    static int CHARACTER_COUNT;
+     String name;
+     int health;
+     int strength;
+     int defence;
+     static int characterCount;
 
+ public Character(String name, int health, int strength, int defence){
+     this.name = name;
+     this.health = health;
+     this.strength = strength;
+     this.defence = defence;
+     characterCount++;
+ }
 
-
-    public Character(){
-        CHARACTER_COUNT++;
-    }
     void attack (Character otherCharacter){
-        if (strength - otherCharacter.defence >= 0) {
-            otherCharacter.health = otherCharacter.health - (strength - otherCharacter.defence);
+        int damage = strength - otherCharacter.defence;
+        if ( damage >= 0) {
+            otherCharacter.takeDamage(damage);
         }
         else {
             otherCharacter.health = otherCharacter.health - 0;
```

Comment:
It doesn't make much sense right? I mean we could remove this `else` block and nothing would change